portal-api/src/v1/api.js

```diff
@@ -450,6 +450,27 @@ const getNextReportPeriodByBankId = async (bankId) => {
   }
 };
 
+/**
+ * Gets the portal facility amendment
+ * @param {string} facilityId - id of the facility to amend
+ * @param {string} amendmentId - id of the facility amendment
+ * @returns {Promise<(import('@ukef/dtfs2-common').PortalFacilityAmendmentWithUkefId)>} - the amendment
+ */
+const getPortalFacilityAmendment = async (facilityId, amendmentId) => {
+  try {
+    const response = await axios({
+      method: 'get',
+      url: `${DTFS_CENTRAL_API_URL}/v1/portal/facilities/${facilityId}/amendments/${amendmentId}`,
+      headers: headers.central,
+    });
```

Comment:
Do we want to validate this as it comes into the service

```
const { success, error, data } = schema.safeParse(response);

if (!success) {
  console.error('a useful error message')
}

return data
```

maybe even abstract this logic into a helper